Source/hydro/ppm.cpp

```diff
@@ -0,0 +1,157 @@
+#include "Castro.H"
+#include "Castro_F.H"
+#include "Castro_hydro_F.H"
+
+#ifdef RADIATION
+#include "Radiation.H"
+#endif
+
+#include <cmath>
+#include <ppm.H>
+
+using namespace amrex;
+
+AMREX_GPU_HOST_DEVICE
+void
+Castro::ppm_reconstruct(const Real* s,
```

Comment:
For optimal GPU performance, we want to be able to inline the PPM reconstruction and integration. So it would be better if these were implemented as `inline` functions in `ppm.H`.

Reply:
done